docs/configuration.rst

```diff
@@ -190,6 +190,20 @@ below:
 
          **Note** that the JSON object must be included in single quotes (') to avoid problems with escaping of the double quote (") character.
 
+   DD_SPAN_SAMPLING_RULES:
+     type: JSON array
```

Comment:
```suggestion
     type: string
```

Reply:
Isn't JSON array correct, since it's the same as `DD_TRACE_SAMPLING_RULES` or does that need to be changed too?

Reply:
the input data type is a string, since you are doing `DD_SPAN_SAMPLING_RULES="<value>"` the fact that it is a JSON string can be covered in the description.